.cirrus.yml

```diff
@@ -57,9 +57,9 @@ task:
   << : *TEST
   i386_test_script:
     - . $HOME/.cargo/env
-    - cargo build --target i686-unknown-freebsd --all-features
```

Comment:
Looks like this was reverted accidentally

Reply:
Hmmm... Good catch. I wonder how I managed that. Time to go to sleep I guess.